.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,28 @@
+# Description
+
+<!---Please include a summary of the change and which issue is fixed. List any dependencies that are required for this change.----->
+
+Fixes:  #issue no.
+
+<!---give the issue number you fixed----->
+
+## Type of change
+
+<!----Please delete options that are not relevant. And in order to tick the check box just put x inside them for example [x] like this----->
+
+- [] Bug fix (non-breaking change which fixes an issue)
+- [] New feature (non-breaking change which adds functionality)
+
+## Checklist:
+
+<!----Please delete options that are not relevant.And in order to tick the check box just put x inside them for example [x] like this----->
+
+- [] I have made this from my own.
```

Comment:
I don't think we require this line, kindly remove it.

Reply:
okay